tasks/agent.py

```diff
@@ -469,6 +469,11 @@ def omnibus_build(
     # base dir (can be overridden through env vars, command line takes precedence)
     base_dir = base_dir or os.environ.get("OMNIBUS_BASE_DIR")
 
+    if sys.platform == 'win32':
+        # On Windows, prevent backslashes in the base_dir path otherwise omnibus will fail with
+        # error 'no matched files for glob copy' at the end of the build.
+        base_dir = base_dir.replace('\\', '/')
```

Comment:
We should probably use the built-ins for separator consts rather than hardcoded separators. What about something like:
```suggestion
        base_dir = base_dir.replace(os.path.sep, '/')
```

WDYT?